cdc/sink/mq.go

```diff
@@ -230,9 +232,32 @@ func (k *mqSink) EmitCheckpointTs(ctx context.Context, ts uint64, _ []model.Tabl
 	if msg == nil {
 		return nil
 	}
-	log.Debug("emit checkpointTs", zap.Uint64("checkpointTs", ts))
-	err = k.writeToProducer(ctx, msg, -1)
-	return errors.Trace(err)
+	// NOTICE: When there is no table sync,
+	// we need to send checkpoint ts to the default topic. T
+	// This will be compatible with the old behavior.
+	if len(tables) == 0 {
+		topic := k.eventRouter.GetDefaultTopic()
+		partitionNum, err := k.topicManager.Partitions(topic)
```

Comment:
`Partitions` is expected to return a partition list on the first sight. It's better to replace with name like `GetPartitionCount`.

Reply:
I refer to sarama's command. If you have a strong opinion about it, I can modify it. 
Thanks for your review!  💚 💙 💜 💛 ❤️